documentation/source/users/rmg/input.rst

```diff
@@ -469,6 +469,22 @@ For example ::
 			  branchingRatioMax=1.0,
 		)
 
+Advanced Setting: Deadend Radical Elimination (RMS Reactors only)
+-----------------------------------------------------------------
+There are many important low-flux pathways that consume radicals, such as chain transfer and chain termination reactions in polymerization.
```

Comment:
My understanding is that the deadend radical algorithm solves the problem that in molecular growth cases with many radicals and fast propagation pathways, non-propagation reactions that consume radicals individually can be so low flux that they do not affect radical concentrations and thus do not register in the flux or branching algorithm, but together control the overall radical loss and thus are very important to include? 

I don't think the use case for the algorithm is made clear here.